addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,234 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
     &.o_kanban_ungrouped {
         .o-justify-content(center);
+        padding: 12px;
+        min-height: 100%;
 
         .o_kanban_record {
-            .o-flex(0, 0, auto);
-            width: 100%;
+            .o-flex(0, 1, auto);
+            width: 50%;
+            margin: 0;
+
+            &:first-child, &:nth-child(2) {
+                @media (max-width: @screen-xs-max) {
+                    margin-top: 10px;
+                }
+            }
+
             @media (min-width: @screen-sm-min) {
-                width: 45%;
+                width: percentage(1/3);
+            }
+
+            @media (min-width: @screen-md-min) {
+                width: percentage(1/4);
             }
         }
     }
 
     .o_kanban_record {
-        padding: 0;
+        padding: 1% 3% 3%;
+        box-shadow: none;
+        background: none;
+        border: none;
+        -webkit-font-smoothing: antialiased;
+
+        .o_browser_bar {
+          background: white;
+          border-bottom: 1px solid darken(@o-theme-kanban-gray, 6%);
+          .size(100%, 20px);
+
+          &:before {
+            content: "•••";
+            display: block;
+            padding-left: 10px;
+            color: darken(@o-theme-kanban-gray, 6%);
+            font: 12px/20px Courier, monospace;
+            letter-spacing: -1px;
+          }
+        }
+
+        &.o_theme_installed .o_theme_preview_top {
+            border-color: @odoo-brand-optional;
+
+            &::after {
+                .o-position-absolute(10px, 10px);
+                .square(20px);
+                display: block;
+                line-height: 20px;
+                text-align: center;
+                background-color: @odoo-brand-optional;
+                border-radius: 100%;
+                font-family: 'FontAwesome';
+                color: white;
+                content: "\f00c";
+            }
+        }
+
+        .o_theme_preview_top {
+            background: white;
+            border: 1px solid darken(@o-theme-kanban-gray, 20%);
+            border-radius: 2px;
+            position: relative;
+            overflow: hidden;
+            will-change: transform;
+            margin-bottom: 5px;
+            .o-transform-origin(center, bottom);
+            .o-transition(all, .1s);
+
+            .o_theme_cover, .o_theme_logo, .o_theme_screenshot {
+                width: 100%;
+                padding-bottom: 127%;
+                background-repeat: no-repeat;
+                background-position: center center;
+                background-color: @gray-lighter;
+                background-size: 40% 32%;
+                .o-transition(opacity, 100ms);
+
+                &.o_theme_cover {
+                    background-size: contain;
+                }
+
+                &.o_theme_screenshot {
+                    background-size: cover;
+                }
```

Comment:
You have rules which are `.o_theme_cover, .o_theme_logo, .o_theme_screenshot`, I think that this should be replaced by something like `.o_theme_image`. Mainly because it is strange to have generated rules which are `.o_theme_cover.o_theme_cover`. So in the xml, where you have the `o_theme_cover, o_theme_logo, o_theme_screenshot` classes, you can add the additional `o_theme_image` class.